### PR TITLE
deb: support to skip lintian

### PR DESCRIPTION
**Pull request by kenhys**
There is a case that lintian check stalls on ubuntu focal
arm64. Until that issue is fixed, skip lintian check as
a workaround.

Closes: #66

### CODE REVIEW

**Comment by kenhys** [comment]
waiting to pass CI

File: lib/apt/build.sh [hunk 1]
```diff
@@ -79,7 +79,11 @@ sed -i'' -E "s/^($PACKAGE \(\S+\)) unstable;/\1 $DISTRIBUTION;/g" debian/changel
 if [ "${DEBUG:-no}" = "yes" ]; then
   run debuild -us -uc --lintian-opts --profile td-agent/${distribution}
 else
-  run debuild -us -uc --lintian-opts --profile td-agent/${distribution} > /dev/null
+  if [ "${LINTIAN:-yes}" = "yes" ]; then
```
Comment (ashie) [inline lib/apt/build.sh:82]: I think lintian check should be done by default, skip it only when a user disables it.

Comment (kenhys) [reply lib/apt/build.sh:82]: "user disables it" means setting LINTIAN=no environment variable by manual,  am I right?

Comment (ashie) [reply lib/apt/build.sh:82]: Right.

Comment (kenhys) [reply lib/apt/build.sh:82]: Thanks, I got it.

**Comment by kenhys** [comment]
Oops, --no-lintian is ignored. I'll check again.

**Comment by ashie** [comment]
> Oops, --no-lintian is ignored. I'll check again.

I've fixed it.

**Pull request merged**